flutter_yaml.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: flutter.yaml Format
+permalink: /flutter_yaml
+---
+
+# {{ page.title }}
+
+Every Flutter application needs some metadata so it can specify its
+dependencies (services, assets, etc.). Flutter stores this information in a file named `flutter.yaml`, which is written in
+the [YAML](http://www.yaml.org/) language.
+
+At the top level are a series of fields. The currently supported ones are:
+
+`name`
+: Required for every application.
+
+`assets`
+: Optional. Relative paths to assets for the application.
+
+`material-design-icons`
+: Optional. Dictionaries specifying icons to include as assets.
+
+`services`
+: Optional. Platform and third-party services (Firebase, Google Cloud Messaging, etc.) to expose to your application.
+
+All other fields are ignored. A simple but complete pubspec looks something
```

Comment:
You're missing fields for the update service:

version: x.y.z
update-url: https://example.com/foo/bar/

see https://github.com/flutter/flutter/wiki/On-the-fly-app-updates